spec/id_generator_spec.rb

```diff
@@ -1,8 +1,10 @@
 require_relative '../lib/inline_svg/id_generator'
 
 describe InlineSvg::IdGenerator do
-  it "generates a hexencoded ID based on a salt" do
-    expect(InlineSvg::IdGenerator.generate("some-base", "some-salt")).
-      to eq("ksiuuy1jduycacqpoj5smn2kyt9iv02")
+  it "generates a hexencoded ID based on a salt and a random value" do
+    randomizer = -> { "some-random-value" }
+
+    expect(InlineSvg::IdGenerator.generate("some-base", "some-salt", randomness: randomizer)).
+      to eq("t2c17mkqnvopy36iccxspura7wnreqf")
```

Comment:
This just happens to be the hex encoded value of "some-base-some-salt-some-random-value".